src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.civil.model.citizenui.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+
+import java.util.Map;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class EventDto {
+
```

Comment:
As I said on the client side, maybe this should be CaseEventDto, with EventDto as an interface?

Reply:
No, I know what DTOs are, I guess that I'm really saying is I'd prefer CaseEventDto as a name as it's more descriptive, given it's transferring a CaseEvent. But if all Events are CaseEvents, and no other sort of Event, then OK.

Reply:
ah cool :) i thought you were proposing an interface for dto which is usually never done with DTO, so i thought perhaps you are not very familiar with it lol.